github/enterprise_codesecurity_configurations.go

```diff
@@ -0,0 +1,201 @@
+// Copyright 2025 The go-github AUTHORS. All rights reserved.
+//
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+package github
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+)
+
+// GetCodeSecurityConfigurations lists all code security configurations available in an enterprise.
+//
+// GitHub API docs: https://docs.github.com/rest/code-security/configurations#get-code-security-configurations-for-an-enterprise
+//
+//meta:operation GET /enterprises/{enterprise}/code-security/configurations
+func (s *EnterpriseService) GetCodeSecurityConfigurations(ctx context.Context, enterprise string) ([]*CodeSecurityConfiguration, *Response, error) {
```

Comment:
Please add query options according to the doc:

<img width="590" height="446" alt="Image" src="https://github.com/user-attachments/assets/a9a4e50a-f010-47be-82b2-517d5999fac8" />

Reply:
I think no. You need to create a struct similar to https://github.com/google/go-github/blob/f508f0dc02caceb5b977fe1f241c7775c7a43c4d/github/projects.go#L58-L67